pvlib/ivtools.py

```diff
@@ -6,6 +6,8 @@
 """
 
 import numpy as np
+from scipy.optimize import root
```

Comment:
The `py35_min` environment purposefully omits scipy. To get the tests to complete, move the `from scipy import …` statements into `fit_sdm_desoto`.  You could include a `try/except` clause like [here ](https://github.com/pvlib/pvlib-python/blob/a873462eb079f7d57cc8c2e5a9cf447ac68027d1/pvlib/solarposition.py#L881) to more gently handle ImportErrors.